client/js/components/statement/originalStatementsTable/OriginalStatementsTableItem.vue

```diff
@@ -241,16 +241,15 @@ export default {
 
     element () {
       let elementTitle = ''
+      const element = this.statement.elementId ? this.elements.find((el) => el.id === this.statement.elementId) : null
 
-      if (hasOwnProp(this.statement, 'elements') && this.statement.elements.title !== '') {
-        elementTitle = this.statement.elements.title
+      if (element) {
+        elementTitle = element.title
```

Comment:
Is there always a title for an element or do we need to check for the title, too?

Reply:
As I see it, elements have parameters such as category, id, ident, and title. Therefore, if the element exists, it must have the title parameter

Reply:
added here https://github.com/demos-europe/demosplan-core/pull/2569/commits/813f451342773422bf0ec88cc85df090e5cb210d

Reply:
Sorry, that was misunderstanding. I meant if we should check for an empty title like we did before `this.statement.elements.title !== ''`

Reply:
ah, sorry done here https://github.com/demos-europe/demosplan-core/pull/2569/commits/cf1574cbc71cca4e2908ab1aebb849f0addf1c97